bin/epaa.py

```diff
@@ -909,6 +901,7 @@ def __main__():
     parser.add_argument('-l', "--max_length", help="Maximum peptide length")
     parser.add_argument('-ml', "--min_length", help="Minimum peptide length")
     parser.add_argument('-t', '--tools', help='Tools used for peptide predictions', required=True, type=str)
+    parser.add_argument('-sv', '--versions', help='File with used software versions.', required=True)
```

Comment:
Should we call it `methods` and maybe `File specifying the applied prediction methods`. We should also be consistent with single and double quotes.

Reply:
I called it `--versions` to distinguish it from  `--tools`. The file contains the whole output from the process `get_software_versions`, independently if a tool specified with `--tool` or not.

Reply:
Changed the quotes (also for `--tools` parameter).